viewer/packages/utilities/src/RandomColors.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import * as THREE from 'three';
+
+/**
+ * Utility class for creating reasonable visible colors. Mainly meant for use
+ * in debugging.
+ */
+export class RandomColors {
```

Comment:
True. Might be we should be remove it, but I think it might become useful. I used this during development and decided to keep it around but I'll remove it if there's any strong feelings.